File: Samples/iOS/iOS/Person+JSON.m
//
//  Person+JSON.m
//  CoreDataClientSample
//
//  Created by Nathaniel Potter on 11/4/14.
//  Copyright (c) 2014 Nathaniel Potter. All rights reserved.
//

#import "Person+JSON.h"

@implementation Person (JSON)

- (void)updateWithDictionary:(NSDictionary *)dictionary
{
    self.uid = dictionary[@"id"];
    self.displayName = dictionary[@"displayName"];
    self.url = dictionary[@"url"];
}

@end

File: Samples/iOS_Swift/CoreDataClientSample/Person+JSON.swift
//
//  Person+JSON.swift
//  CoreDataClientSample
//
//  Created by Nathaniel Potter on 1/31/15.
//  Copyright (c) 2015 Nathaniel Potter. All rights reserved.
//

import Foundation

extension Person {
   
    override func updateWithDictionary(dictionary: [NSObject : AnyObject]!) {
        self.id = dictionary["id"] as? String;
        self.displayName = dictionary["displayName"] as? String;
        self.url = dictionary["url"] as? String;
    }
}